### PR TITLE
Feast tweaks to nav and links

### PR DESCRIPTION
**Pull request by oliverhowell**

### CODE REVIEW

File: docs/modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -181,10 +181,10 @@ include::tiered-storage:partial$nav.adoc[]
 ** xref:spring:transaction-manager.adoc[]
 ** xref:spring:best-practices.adoc[]
 * xref:integrate:integrate-with-feast.adoc[]
-** xref:integrate:install-connect.adoc[]
+** xref:integrate:install-connect.adoc[Install and connect Feast]
 ** xref:integrate:feast-config.adoc[]
-** xref:integrate:feature-engineering-with-feast.adoc[]
-** xref:integrate:streaming-features-with-feast.adoc[]
+** xref:integrate:feature-engineering-with-feast.adoc[Get started with Feast streaming]
```
Comment (yuce) [inline docs/modules/ROOT/nav.adoc:186]:
```suggestion
** xref:integrate:feature-engineering-with-feast.adoc[Get started with Feast batch features]
```

File: docs/modules/ROOT/nav.adoc [hunk 2]
```diff
@@ -181,10 +181,10 @@ include::tiered-storage:partial$nav.adoc[]
...
+** xref:integrate:streaming-features-with-feast.adoc[Get started with Feast feature engineering]
```
Comment (yuce) [inline docs/modules/ROOT/nav.adoc:187]:
```suggestion
** xref:integrate:streaming-features-with-feast.adoc[Get started with Feast streaming features]
```

File: docs/modules/clients/pages/go.adoc [hunk 3]
```diff
@@ -1,6 +1,8 @@
 = Go Client
 :page-api-reference: https://pkg.go.dev/github.com/hazelcast/hazelcast-go-client@v{page-latest-supported-go-client}
 
+TIP: For the latest Go API documentation, see https://pkg.go.dev/github.com/hazelcast/hazelcast-go-client@v{page-latest-supported-go-client}[Hazelcast Go Client docs].
```
Comment (yuce) [inline docs/modules/clients/pages/go.adoc:4]: Omitting the version links to the latest released version.
```suggestion
TIP: For the latest Go API documentation, see https://pkg.go.dev/github.com/hazelcast/hazelcast-go-client[Hazelcast Go Client docs].
```

File: docs/modules/integrate/pages/integrate-with-feast.adoc [hunk 4]
```diff
@@ -115,5 +115,5 @@ To use Feast with Hazelcast, you must do the following:
 
 You can also work through the following tutorials:
 
-* Get Started with Feature Store
-* Feature Compute and Transformation
+* xref:integrate:feature-engineering-with-feast.adoc[Get started with Feast streaming]
```
Comment (yuce) [inline docs/modules/integrate/pages/integrate-with-feast.adoc:118]:
```suggestion
* xref:integrate:feature-engineering-with-feast.adoc[Get started with Feast batch features]
```

File: docs/modules/integrate/pages/integrate-with-feast.adoc [hunk 5]
```diff
@@ -115,5 +115,5 @@ To use Feast with Hazelcast, you must do the following:
...
+* xref:integrate:streaming-features-with-feast.adoc[Get started with Feast feature engineering]
```
Comment (yuce) [inline docs/modules/integrate/pages/integrate-with-feast.adoc:119]:
```suggestion
* xref:integrate:streaming-features-with-feast.adoc[Get started with Feast streaming features]
```

**Review by yuce — COMMENTED** [review]
A few ideas...

File: docs/modules/clients/pages/go.adoc (hunk 3 above)
Comment (oliverhowell) [reply docs/modules/clients/pages/go.adoc:4]: Ooh, nice tip - I might make that a separate PR and back/forward port it as I just cherry picked these changes from 5.4

**Review by yuce — APPROVED** [review]

File: docs/modules/clients/pages/go.adoc (hunk 3 above)
Comment (oliverhowell) [reply docs/modules/clients/pages/go.adoc:4]: Though actually we want this version to point to the one that matches the docs, so the latest supported version should come from the antora file and match that branch e.g. 5.4

**Pull request merged**